pkg/mkverification-raw-efi/vmcap.go

```diff
@@ -0,0 +1,348 @@
+package main
+
+import (
+	"encoding/binary"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const (
+	msrIA32VMXBasic             = 0x480
```

Comment:
Are these defined in some (Intel) document?
Can we refer to a something in a comment?

Reply:
There is information in https://cdrdv2.intel.com/v1/dl/getContent/671200 (Appending A -VMX capability
reporting facility, on page 4483).